docs/_docs/schema.md

```diff
@@ -122,6 +126,10 @@ function(model) {
 }
 ```
 
+**Enum: Array of string**
```

Comment:
This should probably be `enum` not `Enum`, correct? And probably array of strings.

Reply:
Fixed this in my commit to this PR